constellations/orion/src/controllers/family_controller.go

```diff
@@ -0,0 +1,104 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/domains"
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/repos"
+	"github.com/gin-gonic/gin"
+)
+
+func GetAllFamily(c *gin.Context) {
+	// Incoming optional parameter
+	search := c.Query("search")
+	pageSize := ParseParamInt(c.Query("pageSize"), 100)
+	offset := ParseParamInt(c.Query("offset"), 0)
+
+	userList, err := repos.FamilyRepo.SelectAll(search, pageSize, offset)
+	if err != nil {
+		c.Error(err)\
+		c.String(http.StatusInternalServerError, err.Error())
+	} else {
+		c.JSON(http.StatusOK, userList)
+	}
+}
```

Comment:
I think we can get rid of this for now. It makes sense to search by specific users, But not so much for a list of families.